ANMN/burst_averaged_product/burst_average.py

```diff
@@ -148,12 +148,24 @@ def list_var_to_average(netcdf_file_obj):
     var_list = netcdf_file_obj.variables.keys()
     var_list = [x for x in var_list if not x.endswith('_quality_control')]
 
-    for var_to_remove in ['TIME', 'LATITUDE', 'LONGITUDE', 'NOMINAL_DEPTH', 'VOLT', 'TIMESERIES', 'SSPD', 'CNDC']:
-        if var_to_remove in var_list:
-            var_list.remove(var_to_remove)
+    var_to_remove = []
+    for varname in var_list:
+        if 'TIME' not in netcdf_file_obj.variables[varname].dimensions:
+            var_to_remove.append(varname)
+
+    var_to_remove.extend(('TIME', 'VOLT', 'SSPD', 'CNDC'))
```

Comment:
Ok to remove these variables, except TIME. TIME should stay as is. Only data and comment attribute should be updated.